salesforce/10.18/modules/ROOT/pages/salesforce-connector-processing-events.adoc

```diff
@@ -72,6 +72,44 @@ If the application stops and restarts, or if a connection issue triggers a recon
 To persist data so that the app continues processing events when restarted, create a persistent object store, as described in xref:mule-runtime::creating-and-managing-a-cluster-manually.adoc[Create and Manage a Cluster Manually].
 * If you use multiple workers when running the app in CloudHub, all of the workers receive and process the events.
 
+=== Replay Sources Field Configurations
+
+Learn how the priority in replay sources field setup is handled with different configurations:
+
+* *Replay failed events if any or resume from last replay id*
++
+If enabled, this field overwrites anything that is selected in the *Replay option* field and anything that is entered in the *Replay id* field. 
++
+The *Replay failed events if any or resume from last replay id* field checks the ObjectStore for failed event IDs or last known successfully processed event IDs. If there are failed events, processing starts with the lowest failed event, thus receiving all events from `lowestFailedEventId` till now. For example, if `FailedEventIdInObjectStore` is 10 and `LastSuccessfullyProcessedEventIdInObjectStore` is 15, processing starts with 10 and the StreamingClient receives all the events up until 15 and consumes quota. However, these events are not processed again by the flow and are filtered by the connector.
```

Comment:
```suggestion
The *Replay failed events if any or resume from last replay id* field checks the ObjectStore for failed event IDs or last known successfully processed event IDs. If there are failed events, processing starts with the lowest failed event, thus receiving all events from `lowestFailedEventId` until now. For example, if `FailedEventIdInObjectStore` is 10 and `LastSuccessfullyProcessedEventIdInObjectStore` is 15, processing starts with 10 and the StreamingClient receives all the events up until 15 and consumes quota. However, these events are not reprocessed by the flow and are filtered by the connector.
```

Reply:
StreamingClient > streaming client
consumes the quota 

Can you say "the streaming client receives events 15 through 15" - short and sweet?